### ISSUE TITLE
Некорректная зависимость у Visual Studio Extension

### ISSUE THREAD

**Issue by alexeygritsenko** [oleg-shilo/wixsharp#1293]
Добрый день, при установке расширения с https://marketplace.visualstudio.com/items?itemName=OlegShilo.WixSharpProjectTemplates оно потягивает .net framework 4.6.1 targeting pack, в то время как создаваемые расширениями проекты нацелены на .net framework 4.5.1

**решение:** исправить зависимости расширения на .net framework 4.5.1

**Comment by oleg-shilo** [comment]
Sorry, translating from rus does not always help understand the question correctly I suggest you use English so I can accurately interpret the question and others can take benefit from the discussion.

I am not sure I understand "оно потягивает .net framework 4.6.1 targeting pack"

**Comment by alexeygritsenko** [comment]
okey, I'll try. your extension, when installed on a fresh studio, pulls up a higher version of the framework. As a result, template projects do not work until I manually install the correct version.
Maybe this line is the culprit:
https://github.com/oleg-shilo/wixsharp/blob/db13a12df2d705aaa14be7205ab358911387f90f/Source/Templates.2022/WixSharpVSIX/WixSharpVSIX/source.extension.vsixmanifest#L38

**Comment by oleg-shilo** [comment]
Ah, OK, now I see. You were talking about a prerequisite for the project template extension.

Will fix it in the very next release.
Txs

**Comment by oleg-shilo** [comment]
Oh, unfortunately, it is not possible. VSX SDK does not allow setting the extension prerequisite lower than 4.6 even though the extension project template is allowed to target lower versions in the project settings.

![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/d1709289-ce2f-4a6e-a91e-5ba8340dfedd)

Thus it is a clumsy SDK/IDE limitation :(

**Comment by alexeygritsenko** [comment]
ok, thanks for checking this out. well, if so, then so be it. the main thing is that we now know the reason

**Comment by oleg-shilo** [comment]
Agree. Thank you for helping me to check it.

**Comment by alexeygritsenko** [comment]
Huh, today I decided to double check, opened your extension in studio 2019 and there I have a choice
![image](https://github.com/oleg-shilo/wixsharp/assets/10233486/00b54343-3b82-478a-8e37-140c4235c5f4)

**Comment by oleg-shilo** [comment]
Yeah, but in order to build the extension truly targeting VS2022 it needs to be built with... well, VS2022.

In the past, I maintained extensions built on early versions of VSX SDK but still loadable in the newer IDE in the compatibility mode.
This was always a very challenging task that lead to very difficult migration exercises when MS decides to drop forward compatibility on a specific VSX API version.

Thus I have made the decision to target by default the latest IDE version available. Like in this case.
